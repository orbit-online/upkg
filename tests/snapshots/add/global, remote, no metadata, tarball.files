.
├── bin
│   └── orbit-cli -> ../lib/upkg/.upkg/.bin/orbit-cli
└── lib
    └── upkg
        ├── .upkg
        │   ├── .bin
        │   │   └── orbit-cli -> ../orbit-cli-v0.1.3.tar.gz/bin/orbit-cli
        │   ├── orbit-cli-v0.1.3.tar.gz -> .packages/orbit-cli-v0.1.3.tar.gz@1d04b2b7e0d9edbb611bff4ea032de6d5dc98ccc77be6e7d1b51c175da299511
        │   └── .packages
        │       └── orbit-cli-v0.1.3.tar.gz@1d04b2b7e0d9edbb611bff4ea032de6d5dc98ccc77be6e7d1b51c175da299511
        │           └── bin
        │               └── orbit-cli
        └── upkg.json

10 directories, 4 files
