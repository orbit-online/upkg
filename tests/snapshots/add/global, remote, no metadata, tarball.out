upkg: No checksum given for 'https://s3-eu-west-1.amazonaws.com/orbit-binaries/orbit-cli-v0.1.3.tar.gz?AWSAccessKeyId=AKIAZVIOIP7XN4CAKZNT&Expires=2028891302&Signature=03Zofm0v1BcNK%2Bd6RzIlTUwuRsQ%3D', determining now
upkg: Downloading https://s3-eu-west-1.amazonaws.com/orbit-binaries/orbit-cli-v0.1.3.tar.gz?AWSAccessKeyId=AKIAZVIOIP7XN4CAKZNT&Expires=2028891302&Signature=03Zofm0v1BcNK%2Bd6RzIlTUwuRsQ%3D
upkg: Linking 'orbit-cli'
upkg: Added 'https://s3-eu-west-1.amazonaws.com/orbit-binaries/orbit-cli-v0.1.3.tar.gz?AWSAccessKeyId=AKIAZVIOIP7XN4CAKZNT&Expires=2028891302&Signature=03Zofm0v1BcNK%2Bd6RzIlTUwuRsQ%3D'
